tests/system/Database/Builder/InsertTest.php

```diff
@@ -91,9 +91,6 @@ public function testInsertBatch()
         $query = $this->db->getLastQuery();
         $this->assertInstanceOf(Query::class, $query);
 
-        $raw = 'INSERT INTO "jobs" ("description", "id", "name") VALUES (:description:,:id:,:name:), (:description.1:,:id.1:,:name.1:)';
-        $this->assertSame($raw, str_replace("\n", ' ', $query->getOriginalQuery()));
```

Comment:
I guess now `getOriginalQuery` has no data or the result has changed? And that's the reason we had to remove this part of the test? If so, we have to consider this as a BC break change. I understand the goal here - reduce memory consumption and add speed bust - and that's why I'm not against it. But we have to make it clear in the changelog and upgrade notes.

Reply:
Yes, `getOriginalQuery()` returns `INSERT INTO "jobs" ("description", "id", "name") VALUES ('There''s something in your teeth',2,'Commedian'), ('I am yellow',3,'Cab Driver')`. Because we do not use binds.

Reply:
Added docs.